subprojects/unit-system.wrap

```diff
@@ -0,0 +1,6 @@
+[wrap-git]
+url = https://github.com/noah1510/unit-system.git
+revision = v0.5.0
+
+[provides]
+dependency_names = ['unit-system-full','unit-system-generic','unit-system-base','unit-system-common']
```

Comment:
The wrapdb documentation says this section is called `[provide]`, no "s", and that ini values are a comma-separated (and unquoted) list.

As is, this does not currently work at all... it isn't generally ini syntax, it looks toml inspired? But the previous section isn't quoted, anyway. :p

An easy way to check is to add `dependency('unit-system-base')` to the main meson.build and see if the subproject gets required.